promdens.py

```diff
@@ -526,15 +564,11 @@ def windowing(self):
 parser.add_argument("input_file", help="Input file name.")
 
 ### entering code ###
-print("\n##########################################################\n"
-      "###  Promoted density approach for initial conditions  ###\n"
-      "###                   * * * * *                        ###\n"
-      "###       version 1.0         Jiri Janos 2024          ###\n"
-      "##########################################################\n")
-
-# parsing the input and printing it
-print("* Parsing the input.")
```

Comment:
I would keep this line, because the input parameters are plotted after it and without it, it would break a bit the structure. We can have only `print("* Input:")`.

Reply:
Ah, good point, will put it back